### PR TITLE
Refactor awesome books es6

### PR DESCRIPTION
**Pull request by Enning94**
### Awesome-Books refactor using ES6 modules

In this milestone, I:

-Created a new directory called modules and initialized it with git.
-Created the entry point for your JavaScript code called index.js
-Created index.css file for your styles.
-Created the main index.html file.
-Divided my code into modules.
-Refactored my methods and functions to arrow functions.
-Displayed the current date by importing Luxon library as a module.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// eslint-disable-next-line no-unused-vars
```
Comment (elmar8287) [inline index.js]: - [ ] Please, consider removing the linter disables, let the linter makes its job, to find bugs in the first steps of building your project 👍

File: index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+// buttons
+const listBtn = document.querySelector('#list');
+const addBookBtn = document.querySelector('#add-new');
+const contactBtn = document.querySelector('#contact');
+
+// sections
+const contact = document.querySelector('.contact');
+const bookList = document.querySelector('.shelve');
+const addNewBook = document.querySelector('.form');
```
Comment (elmar8287) [inline index.js]: - [ ] Please, consider removing all comments in your code, to make it more clear. Actually, your code is telling us what happening by itself 👍

File: index.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+// eslint-disable-next-line no-unused-vars
+import formInput from './modules/formInput.js';
```
Comment (elmar8287) [inline index.js]: - [ ] Please, consider that this importing is none - there is no such an exporting in the `formInput.js` module. In this case, you can remove this line - it is doing nothing 💯

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @Enning94.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice design
- Nice PR description 👍 

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi @Enning94, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**